### PR TITLE
Add Material Slider and changes since last sync.

### PR DESCRIPTION
**Pull request by nshahan**
Branch with Material Slider examples 
https://github.com/dart-lang/angular_components_example/tree/github-sync

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,40 @@
...
+ * Add Material Slider component.
+ * Remove `visibility.all` from `@Component` annotations. This is potentially
+   BREAKING.
+ * Material Input:
+   * Allow values to be selectable when disabled on all browsers. This is
+     potentially BREAKING.
+   * Pass aria attributes through to the inner input element. Includes material
+     auto suggest.
+ * Material Select: Add aria states/properties to indicate selected/checked in
+   select item component.
```
Comment (chalin) [inline CHANGELOG.md]: I think you meant:
> <del>select</del> item <del>component</del>.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,40 @@
+## 0.9.0-alpha+10
+
+> NOTE: This code is considered production quality, but depends on angular:
+> 5.0.0-alpha+9. The alpha tag represents the evolving nature of the AngularDart
+> API, not code quality (5.0.0-alpha+9 is used in production Google apps).
+
+ * Add Material Slider component.
+ * Remove `visibility.all` from `@Component` annotations. This is potentially
+   BREAKING.
```
Comment (chalin) [inline CHANGELOG.md]: This same item was listed for 0.9.0-alpha+9. Maybe this is meant to say:

> - Remove <ins>some</ins> ...

I see that some remain (e.g. in `lib/material_slider/material_slider.dart`).

File: lib/material_input/material_input.dart [hunk 3]
```diff
@@ -61,6 +61,7 @@ const String materialInputErrorKey = 'material-input-error';
 /// - `multiple` -- Whether the user can enter multiple values, separated by
 ///   commas. This attribute only applies when type = "email", otherwise it is
 ///   ignored.
+/// - `role` -- The role attribute for the input element.
```
Comment (chalin) [inline lib/material_input/material_input.dart]: Qualifying "role" as a "role attribute" doesn't say much :)  How about:

> `role` -- ARIA role for the input element.

File: CHANGELOG.md (hunk 2 above)
Comment (chalin) [inline CHANGELOG.md]: Did you want to mention that some `preserveWhitespace: true` have been removed?

File: lib/material_slider/material_slider.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<!--
+ Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
+ for details. All rights reserved. Use of this source code is governed by a
+ BSD-style license that can be found in the LICENSE file.
+-->
```
Comment (chalin) [inline lib/material_slider/material_slider.html]: Do you really want the copyright notice as a comment inside the generated template? (I can't recall if this is what has been done elsewhere.)

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,10 +1,40 @@
...
+   * Fix transitions.
+   * Simplify material-fab-menu animation & allow it to scroll on small screens.
+ * Material Menu:
+   * Allow mixins to customize popup contents by specifying additional CSS
+     classes added to the popup content.
+ * Material Popup: Enforce space constraints on content.
+ * Protect against a null pointer exception in laminate popup.
+ * Add `SingleSelectionModel` interface with `selectedValue()` method.
+ * Add `onDisabledChanged()` to all `ControlValueAccessor` implementations.
+ * Make `ShowHideDirective` constansts private.
```
Comment (chalin) [inline CHANGELOG.md]: Do you want to mention that as "potentially BREAKING" too?

File: CHANGELOG.md (hunk 2 above)
Comment (nshahan) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 1 above)
Comment (nshahan) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 5 above)
Comment (nshahan) [reply CHANGELOG.md]: Done.

File: lib/material_input/material_input.dart (hunk 3 above)
Comment (nshahan) [reply lib/material_input/material_input.dart]: Leaving this for a future update.

File: CHANGELOG.md (hunk 2 above)
Comment (chalin) [reply CHANGELOG.md]: (Agreed, but from experience, I had to adjust some of the example apps following this switch in default in Angular ... so I just thought that I'd bring it up.)

File: lib/material_slider/material_slider.html (hunk 4 above)
Comment (chalin) [reply lib/material_slider/material_slider.html]: Ah, I thought that I'd seen some .html files w/o that copyright comment, but it seems that I'm mistaken.

**Pull request merged**